packages/components/src/pages/install-guide/InvestigateFindings.vue

```diff
@@ -90,30 +90,27 @@
           </article>
         </div>
         <div v-else data-cy="runtime-analysis-info">
-          <div class="hero columns">
-            <header class="secondary-header">
-              <h1 data-cy="title">AppMap Runtime Analysis</h1>
-              <div class="subheading">
-                Find software design flaws that impact security, performance,
-                stability, and maintainability. Our runtime code analysis can
-                find the problems that static code analyzers miss — and that
-                cause serious production issues.
-              </div>
-            </header>
+          <header>
+            <h1 data-cy="title">AppMap Runtime Analysis Early Access</h1>
+            <div class="subheading">
+              Find software design flaws that impact security, performance,
+              stability, and maintainability. Our runtime code analysis can find
+              the problems that static code analyzers miss — and that cause
+              serious production issues.
+            </div>
+          </header>
+
+          <div class="feature-wrap content">
             <a
-              class="btn-slack"
-              data-cy="slack-button"
-              href="https://appmap.io/slack"
+              class="youtube-video"
+              href="https://appland.com/docs/analysis#demo"
             >
-              <SlackLogo />
-              <v-button
-                label="Join our Slack for early access"
-                class="cta-button b-0"
+              <img
+                class="video-thumbnail"
+                alt="runtime analysis video thumbnail"
+                src="../../assets/runtime_analysis_video_thumbnail.png"
```

Comment:
I believe this is the first `png` we're adding to the distribution. Can you confirm this will properly be added to the bundle that's published to npm?

Reply:
When I locally generate the `dist/index.js` file and use it to load the Extension Host VS Code instance the `png` shows up okay:

![image](https://user-images.githubusercontent.com/45714532/182695099-fd3c0087-4ce2-45db-aed4-ad4d3ef03f71.png)

Should I do anything else to confirm?